tabular/src/autogluon/tabular/models/pecos/run.py

```diff
@@ -0,0 +1,124 @@
+# Run PECOS with AutoGluon. Testing file provided for convenience
+from autogluon.tabular import TabularDataset
+from pecos_model import PecosModel
+import pandas as pd
```

Comment:
Thanks for this! Can you also add unit tests akin to the other models? After addressing my other comments PECOS shouldn't need any special inputs to work (ex: cat_features, text_features, num_features shouldn't be necessary).

Here is an example unit tests for vowpalwabbit that is also a command-line based model interface: https://github.com/awslabs/autogluon/blob/master/tabular/tests/unittests/models/test_vowpalwabbit.py

You should be able to do the same unit test, just replacing VW with PECOS.

Reply:
Added unit tests. Thanks!